src/main/java/com/google/api/codegen/py/PythonGapicContext.java

```diff
@@ -276,6 +294,60 @@ private String returnTypeComment(Method method, PythonImportHandler importHandle
         "\n\nRaises:\n  :exc:`google.gax.errors.GaxError` if the RPC is aborted.");
     return pythonCommon.convertToCommentedBlock(contentBuilder.toString());
   }
+  
+  private String methodSnippet(Method method, PythonImportHandler importHandler) {
+    Interface service = (Interface) method.getParent();
+    List<String> importStrings = new ArrayList<>();
+    importStrings.add(
+        PythonImport.create(
+                ImportType.APP,
+                method.getFile().getProto().getPackage()
```

Comment:
Are you sure that's right for the snippet import statement? I spent some time studying what this does and it pulls the package name from language_settings in the vKit YAML. The other code only seems to use this for determining the path in which to place the generated files.

The code for generating package names and import statements seems to use other sources. I copied `method.getFile().getProto().getPackage()` from `PythonImportHandler`.

As an example, the `pubsub_gapic.yaml` definition in `googleapis` says:

```
language_settings:
  python:
    package_name: gcloud.pubsub
```

However, the generated docs with hand-written snippets (http://pythonhosted.org/gax-google-pubsub-v1/google.pubsub.v1.publisher_api.html) uses the package `google.pubsub.v1.publisher_api`.